packages/labs/context/src/lib/elements/context-provider-element.ts

```diff
@@ -0,0 +1,41 @@
+import {LitElement} from 'lit';
+import {property, customElement} from 'lit/decorators.js';
+import {UnknownContext, ContextType} from '../create-context.js';
+
+import {ContextProvider} from '../controllers/context-provider.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'context-provider': ContextProviderElement<UnknownContext>;
+  }
+}
+
+@customElement('context-provider')
+export class ContextProviderElement<
```

Comment:
I think I'd rather skip this whole element in favor of an element directive that can make any element provide a value to a subtree. The `provide()` ChildPart directive I sketched above could also function as an element directive:

```
html`<div ${provide(loggerContext, logger)}><x-child></x-child><x-child></x-child></div>`
```

Reply:
Agreed, removed.